Ivanova/Menu.js

```diff
@@ -0,0 +1,176 @@
+function Meal() {
+    this.price = 0;
+    this.calories = 0;
+}
+
+function Hamburger(size, stuffing) {
+    Meal.call(this);
+    this.size = size;
+    this.stuffing = stuffing;
+}
+
+Hamburger.prototype = Object.create(Meal.prototype);
```

Comment:
It would be better to have such string after each modification of prototype. 
![image](https://user-images.githubusercontent.com/31970322/52565639-6f206e80-2e18-11e9-97ba-02907af53d82.png)
This way allows you to save "constructor" function. 
This recommendations is relevant for all same situations below.